docs/Quick-Start-Guide/Quickstart.md

```diff
@@ -167,7 +167,7 @@ On all servers:
 From any single server:
 
 ```console
-# gluster volume create gv0 replica 3 server1:/data/brick1/gv0 server2:/data/brick1/gv0 server3:/data/brick1/gv0
+# gluster volume create gv0 replica 3 server1:/data/brick1/gv0 server2:/data/brick1/gv0 server3:/data/brick1/gv0 force
```

Comment:
as all servers are different, in ideal scenario, 'force' is not mandatory. Did you get any errors? What was the error?